packages/nodes-from-anchor/test/v01/typeNodes/OptionTypeNode.test.ts

```diff
@@ -0,0 +1,26 @@
+import { numberTypeNode, optionTypeNode } from '@kinobi-so/nodes';
+import test from 'ava';
+
+import { typeNodeFromAnchorV01 } from '../../../src/index.js';
+
+test('it creates option type nodes', t => {
+    t.deepEqual(typeNodeFromAnchorV01({ option: 'u8' }), optionTypeNode(numberTypeNode('u8')));
+});
+
+test('it creates option type nodes with custom prefixes', t => {
+    t.deepEqual(
+        typeNodeFromAnchorV01({ option: 'u8' }),
+        optionTypeNode(numberTypeNode('u8'), { prefix: numberTypeNode('u64') }),
+    );
+});
```

Comment:
I think we can delete that test for v01. I added a `size` to v00 a while ago so it could also support Shank but let's not add it to v01 so it stays true to the Anchor IDL spec.